### DISCUSSION TITLE
demo: evaluating OPA's Wasm from a single html file

### DISCUSSION THREAD

**Discussion by srenatus** [open-policy-agent/community#98] [Wasm]
This is an exercise in simplicity: evaluating OPA's wasm from a static website, hosted on github, via `npm-opa-wasm`:

It'll pass the windows' fragment hash as _input_ to [`test.rego`](https://github.com/srenatus/opa-wasm-browser/blob/main/test.rego):

:cow: https://srenatus.github.io/opa-wasm-browser/#moo "moo" gives you a _pass_,
indicated by the green background

:stop_sign: Anything else is a _deny_, red background.

## References

- [Open Policy Agent](https://www.openpolicyagent.org/), its [Wasm docs](https://www.openpolicyagent.org/docs/latest/wasm/)
- [npm-opa-wasm](https://github.com/open-policy-agent/npm-opa-wasm), the JS SDK used here

**Comment by anderseknert** [comment]
For anyone else looking, the repo is here https://github.com/srenatus/opa-wasm-browser :)
